marimo/_server/ai/prompts.py

```diff
@@ -94,6 +95,21 @@ def get_refactor_or_insert_notebook_cell_system_prompt(
 ) -> str:
     if cell_code:
         system_prompt = f"Here's a {language} document from a Python notebook that I'm going to ask you to make an edit to.\n\n"
+    elif support_multiple_cells:
+        system_prompt = (
+            "You are an AI assistant integrated into the marimo notebook code editor.\n"
+            "Your goal is to create new cells in the notebook.\n"
+            "You can create multiple cells with different languages. Each cell should be wrapped in backticks.\n"
+            "The user may reference additional context in the form @kind://name. You can use this context to help you with the current task.\n"
+            "You can reference variables from other cells, but you cannot redefine a variable if it already exists.\n"
+            "Immediately start with the following format. Do NOT comment on the code, just output the code itself: \n\n"
+            "```python\n{CELL_CODE}\n```\n\n"
+            '```sql\ndf_name = mo.sql("""{SQL_QUERY}""")\n```\n\n'
+            '```markdown\nmo.md("""{MARKDOWN_CONTENT}""")\n```\n\n'
```

Comment:
i think you just want:

````
```sql{SQL_QUERY}```
// and
```markdown{MARKDOWN_CONTENT}```
````

or 

````
'```python\ndf_name = mo.sql("""{SQL_QUERY}""")\n```\n\n'
'```python\nmo.md("""{MARKDOWN_CONTENT}""")\n```\n\n'
````

but not the mix of them, right?